.circleci/config.yml

```diff
@@ -78,6 +78,23 @@ jobs:
       - run: { name: 'Release S3', command: "cd patternlab/styleguide && node_modules/.bin/gulp patternlab:release" }
       - run: { name: 'Release NPM', command: "cd patternlab/styleguide && node_modules/.bin/gulp npm:release" }
 
+  react_build:
+    working_directory: /var/www/code
+    docker:
+      - image: circleci/node:8.9.4-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - mf-react-dependencies-{{ checksum "package.json" }}
+      - run: { name: 'NPM Install', command: 'cd react && npm install' }
+      - save_cache:
+          name: Save NPM cache
+          key: mf-react-dependencies-{{ checksum "package.json" }}
+          paths:
+            - node_modules
```

Comment:
Each step executes in a new shell independent of the last. So (kinda crazy), but you have to `cd react` at each step.  In this case `paths` is relative to the root, so it should be `react/node_modules`.  Ugh.. I just realized that this is why our caching isn't working for the Patternlab build.